### PR TITLE
feature/MBA-48_create-frequently-used-functions

### PR DESCRIPTION
**Pull request by JeriRov**
201802319-3fef59fb-a220-4c4a-b4e0-a4cc97e48693

### CODE REVIEW

File: src/components/FrequentlyUsedFunctions/FrequentlyUsedButtons/FrequentlyUsedButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    onPress: depositButtonOnPress,
+  },
+  {
+    title: detailsButtonTitle,
+    iconName: detailsButtonIcon,
+    onPress: detailsButtonOnPress,
+  },
+];
+
+const FrequentlyUsedButton = (props: IFrequentlyUsedButtonProps) => {
```
Comment (milkanuch) [inline src/components/FrequentlyUsedFunctions/FrequentlyUsedButtons/FrequentlyUsedButtons.tsx]: Put this in other file tsx file

File: src/components/FrequentlyUsedFunctions/FrequentlyUsedButtons/FrequentlyUsedButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  detailsButtonOnPress,
+  detailsButtonTitle,
+  iconColor,
+  iconSize,
+  transferButtonIcon,
+  transferButtonOnPress,
+  transferButtonTitle,
+} from './frequentlyUsedButtons.settings';
+
+const buttons = [
```
Comment (milkanuch) [inline src/components/FrequentlyUsedFunctions/FrequentlyUsedButtons/FrequentlyUsedButtons.tsx]: and this is in the buttons settings

**Review by milkanuch — CHANGES_REQUESTED** [review]

**Review by milkanuch — APPROVED** [review]
LGTM

**Review by stepan103ua — APPROVED** [review]
LGTM

**Pull request merged**